README.md

```diff
@@ -92,13 +92,12 @@ _Includes the `RBAC_URL` and `APPROVAL_URL environment variables to be able to u
 _You can also remove the `RBAC_URL` environment variable and pass in the `BYPASS_RBAC` environment variable instead if you want to test without RBAC_
 
 ```
-   APP_NAME=catalog PATH_PREFIX=/api RBAC_URL=https://<rbac_url>/api/rbac/v1/ APPROVAL_URL=https://<approval_url>/api/approval/v1/ DEV_USERNAME=<username> DEV_PASSWORD=<password> bin/rails s -
-p 5000
+APP_NAME=catalog PATH_PREFIX=/api RBAC_URL=https://<rbac_url>/api/rbac/v1/ APPROVAL_URL=https://<approval_url>/api/approval/v1/ TOPOLOGICAL_INVENTORY_URL=https://<topology_url>/api/topological-inventory/v1/ SOURCES_URL=https://<sources_url>/api/sources/v1/ DEV_USERNAME=<username> DEV_PASSWORD=<password> bin/rails s -p 5000
```

Comment:
I wonder if <rbac_url>, <approval_url> etc should be modified to just <url> since the base url for all the services within a dev environment is the same?

Reply:
Updated to use `<url>` for all.